alicloud/table_alicloud_account.go

```diff
@@ -0,0 +1,96 @@
+package alicloud
+
+import (
+	"context"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ram"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudAccount(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_account",
+		Description: "Alicloud Account",
+		List: &plugin.ListConfig{
+			Hydrate: listAccountAlias,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "alias",
```

Comment:
Is this the only property we get back about the account?

Reply:
@cbruno10 Currently the API returns only `AccountAlias` in response
https://www.alibabacloud.com/help/doc-detail/28737.htm